.github/workflows/integration_tests_v2.yml

```diff
@@ -0,0 +1,41 @@
+name: integration-tests-v2
+
+on: push
+
+jobs:
+  integration-tests:
```

Comment:
Identical to integration_tests.yml (integration test for v1), just with v2 working directory, and v2 in the node module cache